apps/researcher/src/app/[locale]/communities/[id]/page.tsx

```diff
@@ -0,0 +1,107 @@
+import {ChevronLeftIcon} from '@heroicons/react/24/solid';
+import {clerkClient, auth} from '@clerk/nextjs';
+import Link from 'next/link';
+import Image from 'next/image';
+import {getTranslations} from 'next-intl/server';
+import {JoinCommunityButton, EditCommunityButton} from './buttons';
+
+interface Props {
+  params: {id: string};
+}
+
+export default async function CommunityPage({params}: Props) {
+  const t = await getTranslations('Community');
+
+  let organization;
+  try {
+    organization = await clerkClient.organizations.getOrganization({
+      organizationId: params.id,
+    });
+  } catch (error) {
+    return <div data-testid="no-entity">{t('noEntity')}</div>;
+  }
+
+  const memberships =
+    await clerkClient.organizations.getOrganizationMembershipList({
```

Comment:
Could the call to `getOrganizationMembershipList()` fail, and if so, should we catch/handle it here, or can the error bubble up?